flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -1320,8 +1339,46 @@ class ScalarExprLowering {
 
     for (const auto &arg : caller.getPassedArguments()) {
       const auto *actual = arg.entity;
-      if (!actual)
-        TODO("optional argument lowering");
+      if (!actual) {
+        // absent optional.
```

Comment:
Not sure what this comment implies.

Reply:
Changed to `// Optional dummy argument for which there is no actual argument.`